feature/qos/ate_tests/one_sp_queue_traffic_test/one_sp_queue_traffic_test.go

```diff
@@ -777,38 +777,70 @@ func ConfigureQoS(t *testing.T, dut *ondatra.DUTDevice) {
 	d := &oc.Root{}
 	q := d.GetOrCreateQos()
 
+	type qosVals struct {
+		be0, be1, af1, af2, af3, af4, nc1 string
+	}
+
+	qos := qosVals{
+		be0: "BE0",
+		be1: "BE1",
+		af1: "AF1",
+		af2: "AF2",
+		af3: "AF3",
+		af4: "AF4",
+		nc1: "NC1",
+	}
+
+	if dut.Vendor() == ondatra.JUNIPER {
```

Comment:
We shouldn't vary the test expectations without a deviation

Reply:
as stated in pull request 1378, this is not a deviation.